code/style.css

```diff
@@ -1,7 +1,309 @@
+  height: 50vh;
+  background-image: url('/images/hero.jpg');
+  background-size: cover;
+  background-repeat: no-repeat;
+  background-position: center;
+  display: flex;
+  flex-direction: column;
+  font-family: 'Sedgwick Ave', cursive;
+}
+
+.image-header .fab {
+  color:rosybrown;
+  padding: 2px;
+  cursor:pointer;
+} 
+
+.header-title {
+  font-size: 36px;
+  display: flex;
+  justify-content: center;
+}
+
+.header-title a {
+  color:mediumpurple;
+}
+
+.inspo {
+  display:flex;
+  justify-content: center;
+  text-align: center;
+  font-size: 16px;
+  color: black;
+}
+
+.inspo:hover {
+  transform: scale(1.4);
+  cursor: pointer;
+}
+
+.header-text {
+  font-size: 14px;
+  color:mediumvioletred;
+  transition: transform .6s;
+  cursor: pointer;
+  display: flex;
+  justify-content:center;
+  padding: 24px;
+}
+
+.header-text:hover {
+  color:yellow;
+  transform: scale(1.2);
+}
+
+/* --- PARENT TO FORM-CONTAINER and ACCORDION --- */
+.parent {
+  display:flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+
+/* --- STYLING OF THE FORM --- */
+.form-container {
+  width: 300px;
+  height: auto;
+  margin-top: 24px;
+  padding: 24px;
+  border: 2px solid;
+  box-shadow: 8px 8px 0 0 lightgray;
+  background:lavenderblush;
+  border-radius: 4px;
+}
+
+.form-container h2 {
+  font-size: 36px;
+  text-align: center;
+  text-transform: uppercase;
+}
+
+.form-container h2:hover {
+  color:palevioletred;
+  cursor: pointer;
+}
+
+.checkboxes {
+  font-size: 10px;
+  font-family:'Gill Sans', 'Gill Sans MT', Calibri, 'Trebuchet MS', sans-serif;
+}
+  
+.submit-btn {
+  background:pink;
+  border: 2px solid;
+  cursor: pointer;
+  width: 96px;
+  padding: 8px;
+  border-radius: 4px;
+  font-family:Impact, Haettenschweiler, 'Arial Narrow Bold', sans-serif;
+  font-size:24px;
+}
+
+textarea {
+  border: 2px solid;
+  background-color:linen;
+  font-size: 10px;
+  border: 2px solid;
+  border-radius: 4px;
+}
+
+.experience-options {
+  font-family:'Gill Sans', 'Gill Sans MT', Calibri, 'Trebuchet MS', sans-serif;
+  font-size: 12px;
+}
+
+/* --- STYLING OF THE ACCORDION --- */
+.accordion {
+  width: 300px;
+  height: auto;
+  margin-top: 24px;
+  padding: 8px;
+  border-radius: 4px;
+  background:lavenderblush;
+  border: 2px solid;
+  box-shadow: 8px 8px 0 0 lightgray;
+}
+
+.accordion h2 {
+  font-size: 36px;
+  text-align: center;
+}
+
+.accordion h2:hover {
+  color:skyblue;
+  cursor: pointer;
+}
+
+.accordion .contentBx {
+  position: relative;
+  margin: 10px 20px;
+}
+
+.accordion .contentBx a {
+  color:mediumpurple;
+}
+
+.accordion .contentBx .label {
+  position: relative;
+  padding: 10px;
+  background:palevioletred;
+  color: white;
+  cursor: pointer;
+  font-family:Impact, Haettenschweiler, 'Arial Narrow Bold', sans-serif;
+  border-radius: 4px;
+  font-size:12px;
+}
+
+.accordion .contentBx .label:hover {
+  background:rosybrown;
+}
+
+.accordion .contentBx .label::before {
+  content: '+';
```

Comment:
What does this + mean? *curious*

Reply:
This creates the + sign in the accordion :)